index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
Ah Node v4 [is on the decline](https://twitter.com/seldo/status/838922927735132160), something to think about for sure. The only reason I'm holding on is because v4 is LTS.

![](https://cloud.githubusercontent.com/assets/4303/24022410/803ec71e-0a63-11e7-92a7-f6dc8cd75d95.png)